### DISCUSSION TITLE
EMP v2 使用 & 项目开发 QA 总汇

### DISCUSSION THREAD

**Discussion by ckken** [empjs/emp#277] [Q&A]
与 emp v2 相关问题 QA

**Comment by bitem-cn** [comment]
![image](https://user-images.githubusercontent.com/96805327/172272422-05a1c01f-3349-4738-90ce-3cd992b3ed46.png)
有人碰到过这么奇怪的问题吗？使用emp调用远程模块会丢失部分额标签。

**Reply by ckken** [reply]
shared  vue 设置了吗
